packages/ui/src/components/date-input/date-input.test.tsx

```diff
@@ -0,0 +1,27 @@
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import DateInput from "./date-input";
+
+describe("Default Date Input and click input trigger", () => {
+  const user = userEvent.setup();
+  beforeEach(async () => {
+    render(
+      <div>
+        <DateInput>From</DateInput>
```

Comment:
Can we get a test for child prop? Kent Dodds says,

 [The more your tests resemble the way your software is used, the more confidence they can give you.]
(https://twitter.com/kentcdodds/status/977018512689455106)

Text is critical to the user experience. We want to make sure text renders correctly in the future!

Reply:
Done!